tests/functional/object/test_remote_obc_crud.py

```
@@ -516,3 +521,345 @@ def _calculate_sha256(self, file_path):
+            logger.info("Step 10: Verifying reads continue with one mirror down")
+
+            with config.RunWithConfigContext(client_index):
+                # Attempt to read object - should succeed from remaining mirror
+                try:
+                    response = s3_client.get_object(
+                        Bucket=bucket_name, Key=test_object_key
+                    )
+                    data = response["Body"].read()
+                    assert len(data) == len(test_data), "Data size mismatch"
+                    logger.info("Read operation successful with one mirror down")
+                except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)